ADAL/src/validation/ADAuthorityValidation.m

```diff
@@ -381,12 +381,12 @@ - (void)validateADFSAuthority:(NSURL *)authority
 }
 
 - (void)requestDrsDiscovery:(NSString *)domain
-                    context:(id<MSIDRequestContext>)context
+              requestParams:(ADRequestParameters *)requestParams
```

Comment:
Just curious, any reason it needs to be `ADRequestParameters` but not `id<MSIDRequestContext>`? :)

Reply:
Nope, it got merged from 2.6.6 :) Fixed it.